src/components/MediaContent/ButtonText/ButtonText.component.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { StyledButton } from './ButtonText.styles';
+
+const ButtonText = ({ handleClick, expandTextButton }) => {
+  return (
+    <StyledButton onClick={handleClick}>
+      {expandTextButton ? <span>contract</span> : <span>expand</span>}
```

Comment:
Creo que podríamos simplificar esto a
```suggestion
      <span>{ expandTextButton ? 'contract' : 'expand' }</span>
```